Management.Database/Stores/ManaSqlVoteStore.cs

```diff
@@ -0,0 +1,104 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Common;
+using Common.Exceptions;
+using Dapper;
+using Management.Database.Interfaces;
+using Management.Database.Models;
+using Management.Models;
+
+namespace Management.Database.Stores
+{
+    public class ManaSqlVoteStore : SqlStore, IManaVoteStore
+    {
+        public ManaSqlVoteStore(string connectionString) : base(connectionString)
+        {
+        }
+
+        private async Task<IEnumerable<ManaVote>> GetVoteByQuery(string query, object parameters)
+        {
+            try
+            {
+                using (var connection = await GetConnectionAsync())
+                {
+                    var result = await connection.QueryAsync(query, parameters);
+
+                    if (!result.Any())
+                        throw new RecordNotFoundException();
+
+                    return result.Select((v) => new ManaDbVote(v)).Select((v) => v.ToVote());
```

Comment:
no need to iterate through twice. Just `Select(v => new ManaDbVote(v).ToVote());`